apps/src/templates/projects/ProjectWidget.jsx

```diff
@@ -20,12 +20,42 @@ const ProjectWidget = React.createClass({
         isRtl={false}
       >
         {this.props.projectList.length > 0 &&
-          <PersonalRecentProjects projectList={this.props.projectList} />
+          <PersonalRecentProjects
+            projectList={convertChannelsToProjectData(this.props.projectList)}
+          />
         }
-        <NewProjectButtons/>
+        <NewProjectButtons projectTypes={this.props.projectTypes}/>
       </ContentContainer>
     );
   }
 });
 
+// The project widget uses the channels API to populate the personal projects
+// and the data needs to be converted to match the format of the project cards
+// before passing it to PersonalRecentProjects.
+const convertChannelsToProjectData = function (projectLists) {
+  // Sort by most recently updated.
+  projectLists.sort((a, b) => {
+    if (a.updatedAt < b.updatedAt) {
+      return 1;
+    } else {
+      return -1;
+    }
+  });
```

Comment:
I think you could use lodash, and just do `return _.sortBy(projects, 'updatedAt')` while will (a) return a new array and (b) mean you dont need your own sort function